app/src/main/java/com/github/se/wanderpals/model/viewmodel/CreateSuggestionViewModel.kt

```diff
@@ -22,6 +22,14 @@ open class CreateSuggestionViewModel(tripsRepository: TripsRepository) : ViewMod
     return a
   }
 
+  open fun updateSuggestion(tripId: String, suggestion: Suggestion): Boolean {
+    var a: Boolean = true

```

Comment:
`a` is not explicit. Maybe something as "success" could be better to describe what the variable represent